src/base/bittorrent/session.h

```diff
@@ -50,7 +50,7 @@
 #include "sessionstatus.h"
 #include "torrentinfo.h"
 
-#if ((LIBTORRENT_VERSION_NUM >= 10206) && !defined(Q_OS_WIN))
+#if !defined(Q_OS_WIN)
```

Comment:
1.2.12 of libtorrent will load certificates from the Windows store, thus allowing it on all platforms, so we could remove this macro and just change the check to 1.2.12. See https://github.com/arvidn/libtorrent/pull/5313. Alternatively, that could be extracted to a seperate commit.

Reply:
I think this is a separate task. It will need to be tested first before we add support for it.
In addition, if this feature is important, we will simply raise the minimum supported version of libtorrent (once libtorrent 1.2.12 is released and tested) without any conditionals.